bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/GhostAsterism.scala

```diff
@@ -59,9 +59,19 @@ object GhostAsterism {
   final case class GhostTarget(spTarget: SPTarget,
                                guideFiberState: GuideFiberState) {
 
+    /** Target coordinates, not PM corrected. */
     def coordinates(when: Option[Instant]): Option[Coordinates] =
       spTarget.getCoordinates(when.map(_.toEpochMilli))
 
+    /** Target coordinates, PM corrected. */
+    def pmCorrectedCoordinates(when: Option[Instant]): Option[Coordinates] = {
+      val corrected = for {
+        pm <- spTarget.getProperMotion
+        w  <- when
+      } yield pm.calculateAt(spTarget.getTarget, w)
+      corrected orElse coordinates(when)
```

Comment:
Isn't `pm.calculateAt` doing the same as `getCoordinates(when)`?

Reply:
You might think so, but no we don't apply proper motion despite asking for the `when` parameter.   It is ignored except for non-sidereal targets.